src/app/pages/dashboard/components/widget-help/widget-help.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import { MediaObserver } from '@angular/flex-layout';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { TranslateService } from '@ngx-translate/core';
+import { ProductType } from 'app/enums/product-type.enum';
+import helptext from 'app/helptext/about';
+import { WidgetComponent } from 'app/pages/dashboard/components/widget/widget.component';
+import { SystemGeneralService } from 'app/services';
+import { LocaleService } from 'app/services/locale.service';
+
+@UntilDestroy()
+@Component({
+  selector: 'widget-help',
+  templateUrl: './widget-help.component.html',
+  styleUrls: ['./widget-help.component.scss'],
+})
+export default class WidgetHelpComponent extends WidgetComponent implements OnInit {
+  copyrightYear = this.localeService.getCopyrightYearFromBuildTime();
+  extraMsg: boolean;
+  systemType: ProductType;
+  helptext = helptext;
+  readonly ProductType = ProductType;
+  screenType = 'Desktop'; // Desktop || Mobile
+
+  constructor(
+    private localeService: LocaleService,
+    public mediaObserver: MediaObserver,
+    private sysGenService: SystemGeneralService,
+    public translate: TranslateService,
+  ) {
+    super(translate);
+    mediaObserver.media$.pipe(untilDestroyed(this)).subscribe((evt) => {
+      const st = evt.mqAlias == 'xs' ? 'Mobile' : 'Desktop';
+      this.screenType = st;
+    });
+  }
+
+  ngOnInit(): void {
+    this.sysGenService.getProductType$.pipe(untilDestroyed(this)).subscribe((productType) => {
+      this.systemType = productType as ProductType;
+    });
+  }
+
+  turnOffWelcomeDialog(): void {
```

Comment:
Good work!
Image in ticket
![image](https://user-images.githubusercontent.com/22980553/149943913-57a5e676-c8d6-4f51-836b-2c9e38ea8f7a.png)

Web UI
![Screenshot_7](https://user-images.githubusercontent.com/22980553/149943569-16f0cdb8-6e99-4530-b4cd-1d5d48138a7e.png)

![Screenshot_8](https://user-images.githubusercontent.com/22980553/149943603-de7f16c8-0ac9-4a9e-be42-0a75550f918d.png)

There is some difference in UI. Not sure to add a logo like the Welcome dialog.
Missing unit testing.
Please remove all prev codes of the Welcome dialog.